### ISSUE TITLE
Config is always invalid

### ISSUE THREAD

**Issue by litetex** [SpaceWalkerRS/alternate-current#42]
When running this mod I constantly get
```
[Server thread/INFO]: skipping unknown option 'enabledtrueupdate-order' in Alternate Current config
```

I think this is because the config is incorrectly written here:
https://github.com/SpaceWalkerRS/alternate-current/blob/77c6422398b19d29e65bf15104ddc91e462ce47a/src/main/java/alternate/current/wire/Config.java#L121-L127

**Comment by litetex** [comment]
A basic fix is available via #43

Feel free to check it out :)

**Comment by SpaceWalkerRS** [comment]
well, that is embarrassing - I'll push out a new release real quick
